.storybook/manager.ts

```diff
@@ -1,7 +1,8 @@
 import { addons } from '@storybook/addons';
 
-import blackbaudTheme from './blackbaud';
+import { blackbaud } from './blackbaud';
 
 addons.setConfig({
-  theme: blackbaudTheme,
+  // @ts-ignore
+  theme: blackbaud,
```

Comment:
Instead of `@ts-ignore` use `theme: blackbaud as Required<ThemeVars>`.

Reply:
Done